src/http/ngx_http_mruby_module.c

```diff
@@ -2474,7 +2524,11 @@ static int ngx_http_mruby_ssl_cert_handler(ngx_ssl_conn_t *ssl_conn, void *data)
   mrb = mscf->state->mrb;
   mrb->ud = mscf;
   ai = mrb_gc_arena_save(mrb);
-  mrb_run(mrb, mscf->ssl_handshake_code->proc, mrb_top_self(mrb));
+  if (mscf->ssl_handshake_code != NGX_CONF_UNSET_PTR) {
+    mrb_run(mrb, mscf->ssl_handshake_code->proc, mrb_top_self(mrb));
+  } else {
+    mrb_run(mrb, mscf->ssl_handshake_inline_code->proc, mrb_top_self(mrb));
+  }
```

Comment:
When don't use code cache (arg 3 = "cache"), we should recompile the ruby code, like the following:

ref: src/http/ngx_http_mruby_module.c 
```
1879     if (!code->cache) {                                                                                                \
1880       NGX_MRUBY_STATE_REINIT_IF_NOT_CACHED(mlcf->cached, mmcf->state, code, ngx_http_mruby_state_reinit_from_file);    \
1881     }                                                                                                                  \
                                                                                                            \                                                                                                      
```